libs/category/testing/src/inmemory-backend/category.service.ts

```diff
@@ -21,6 +22,7 @@ export class DaffInMemoryBackendCategoryService implements InMemoryDbService {
     private productInMemoryBackendService: DaffInMemoryBackendProductService
   ) {
     this.category = this.categoryFactory.create();
+    this.category.id = '1';
```

Comment:
This should be the partial param in the factory `create`

Reply:
turns out I don't need to set the category Id right now anyway, so I just removed that line